rts/Lua/LuaVBOImpl.cpp

```diff
@@ -1131,16 +1302,42 @@ int LuaVBOImpl::BindBufferRangeImpl(GLuint bindingIndex,  const sol::optional<in
 	return result ? bindingIndex : -1;
 }
 
+
+/***
+ *
+ * @function VBO:BindBufferRange
+ * @tparam number index
```

Comment:
index should be in the range between `5 < index < GL_MAX_UNIFORM_BUFFER_BINDINGS value (usually 31)`